examples/gatekeeper/required-tags/template.yaml

```diff
@@ -8,31 +8,18 @@ spec:
       names:
         kind: AwsRequiredTags
       validation:
-        # Schema for the `parameters` field
         openAPIV3Schema:
           type: object
           properties:
-            tags:
-              type: array
-              items:
-                type: string
+            tag:
+              type: string
```

Comment:
why only one tag? it should be list of k/v pair of tags

Reply:
Fix pushed